lib/pengine/common.c

```diff
@@ -240,40 +245,45 @@ static pcmk__cluster_option_t pe_opts[] = {
     /* Node health */
     {
         PCMK__OPT_NODE_HEALTH_STRATEGY, NULL, "select",
-        "none, migrate-on-red, only-green, progressive, custom",
-        "none", check_health,
+        PCMK__VALUE_NONE ", " PCMK__VALUE_MIGRATE_ON_RED ", "
+            PCMK__VALUE_ONLY_GREEN ", " PCMK__VALUE_PROGRESSIVE ", "
+            PCMK__VALUE_CUSTOM,
+        PCMK__VALUE_NONE, check_health,
         "How cluster should react to node health attributes",
         "Requires external entities to create node attributes (named with "
-            "the prefix \"#health\") with values \"red\", \"yellow\" or "
-            "\"green\"."
+            "the prefix \"#health\") with values \"" PCMK__VALUE_RED "\", "
+            "\"" PCMK__VALUE_YELLOW "\", or " "\"" PCMK__VALUE_GREEN "\"."
```

Comment:
I think you've got a few too many quotes after the "or".  I think that bit can just be `, or \""`

Reply:
good catch, updated commit